my_profiles/example_global_context/builds.yaml

```diff
@@ -0,0 +1,56 @@
+
+## This YAML file is sparsely commented, with a focus on the parts relevant to multiple inputs
+## See my_profiles/example/builds.yaml for more general comments
+## See docs/multiple_inputs.md for a walkthrough of this config.
+
+# custom_rules:
+#   - my_profiles/example_multiple_inputs/rules.smk
+
+use_nextalign: true
+
+inputs:
+  - name: aus
+    metadata: "data/example_metadata_aus.tsv"
+    sequences: "data/example_sequences_aus.fasta"
+  - name: global
+    # Define local paths to a pre-filtered global context.
+    metadata: "data/global_subsampled_metadata.tsv.gz"
+    filtered: "data/global_subsampled_sequences.fasta.gz"
+    # Paths to files on S3 also work. For example:
+    #metadata: "s3://nextstrain-ncov-private/global_subsampled_metadata.tsv.xz"
```

Comment:
Files currently on S3 are `gz` and not `xz`.

Reply:
Good catch! The new subsampled files will have the `.xz` extension, since [the upload rule uses this compression for sequence data](https://github.com/nextstrain/ncov/blob/3ef559a1e975a1ef10cc301b87b056308ac74d89/workflow/snakemake_rules/export_for_nextstrain.smk#L180-L184), too. You can see example outputs from the trial build run here:

```bash
$ aws s3 ls s3://nextstrain-ncov-private/trial/99d07b91a09eb3d05e220e93aca0e84dd0f09bbe/
2021-04-01 14:39:26   70963528 aligned_gisaid.fasta.xz
2021-04-01 14:48:42   43403016 filtered_gisaid.fasta.xz
2021-04-01 14:42:35     218436 flagged-sequences_gisaid.tsv.xz
2021-04-01 14:51:34     202784 global_subsampled_metadata.tsv.xz
2021-04-01 14:51:33     308348 global_subsampled_sequences.fasta.xz
2021-04-01 14:44:17   50371144 masked_gisaid.fasta.xz
2021-04-01 14:51:22   27834876 mutation_summary_gisaid.tsv.xz
2021-04-01 14:42:29   19595104 sequence-diagnostics_gisaid.tsv.xz
2021-04-01 14:42:35     100472 to-exclude_gisaid.txt.xz
```

Moving forward, I'd really like us to compress our sequences with bgzip so we can use samtools faidx (benefits of bgzip indexing outweighing benefits of slightly better compression from LZMA). The bgzip compression would produce a `.gz` extension for all of these uploaded files instead.